sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/plans/PlanTest.scala

```diff
@@ -26,6 +26,9 @@ import org.apache.spark.sql.catalyst.util._
  * Provides helper methods for comparing plans.
  */
 abstract class PlanTest extends SparkFunSuite with PredicateHelper {
+
+  System.setProperty("spark.testing", "true")
```

Comment:
`SPARK_TESTING` is an env variable while `spark.testing` is a JVM system property, but yes both are set by the build for you.